en/SUMMARY.md

```diff
@@ -103,6 +103,7 @@
   * [Helicopter (experimental)](frames_helicopter/README.md)
     * [Helicopter Config/Tuning](config_heli/README.md)
   * [Rovers (experimental)](frames_rover/README.md)
+    * [Aion Robotics R1](frames_rover/aion_r1.md)
```

Comment:
FYI only, needs to go into SUMMARY.md to appear in the sidebar

Reply:
Thanks! I'll keep this in mind for next time.